backpack/core/derivatives/avgpoolnd.py

```diff
@@ -151,6 +169,8 @@ def __apply_jacobian_t_of(self, module, mat):
         elif self.N == 3:
```

Comment:
Refactoring suggestion to avoid the `NotImplementedError`:
```python
output_size = (V_N_C_in, C_for_conv_t) + tuple(module.input0.shape[2:])
```

Reply:
Worked out.